js/modules/cohortbuilder/components/CohortExpressionEditorTemplate.html

```diff
@@ -204,5 +204,20 @@
 				No censoring events selected.
 			</div>
 		</div>	
-	</div>	
+	</div>
+
+	<div role="tabpanel" data-bind="css: { active: $component.expressionMode() == 'exit' || $component.expressionMode() == 'all'}" class="tab-pane">
+		<div class="heading">
+			<b>Cohort Collapse Strategy</b>
+			This strategy collapses events into eras of a continous period and eliminates the duplicate rows for the same subjects.  
+		</div>
+		<div style="padding: 5px 0px" data-bind="with: expression().CollapseSettings">
+			<ul>
+				<li>
+					Specify era collapse gap size: 
+					<select data-bind="options: $component.options.dayOptions, value: EraPad"></select> days
```

Comment:
The input type for a numeric input isn't a select, rather it's done this way:

```
<span contenteditable="true" class="numericInputField dropdown" data-bind="htmlValue: EraPad.numeric(), eventType: 'blur', ko_autocomplete: { value: EraPad, source: $component.options.dayOptions, minLength: 0, maxShowItems: 10, scroll: true }" />
```

Could you modify the <select> element to be the <span> element that's content editable?